recipes-domd/agl/files/meta-xt-prod-extra/recipes-connectivity/connman/connman_%.bbappend

```diff
@@ -0,0 +1,13 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI += "file://main.conf"
+
+FILES_${PN} += " \
+    ${sysconfdir}/main.conf \
```

Comment:
is this a reserved name "main"? or can we have something more descriptive?

Reply:
the name of the configuration file is taken as documentation says